src/UI/NavigationContainer.elm

```diff
@@ -434,13 +437,37 @@ Clicking on the black layer also activates the closing message.
 -}
 dialog : String -> msg -> Element msg -> Dialog msg
 dialog title onClose body =
-    Dialog
+    Dialog1
```

Comment:
Here it would become:
```
        Dialog2.Dialog
            { title = title
            , icon = Nothing
            }
            { buttons = []
            , body = body
            }
````

Reply:
I would add to the docs comment saying this deprecated we must use the other one